maps/torch/torch_ranks.dm

```diff
@@ -4,7 +4,7 @@
 	allowed_branches = list(/datum/mil_branch/civilian)
 	allowed_ranks = list(/datum/mil_rank/civ/civ)
 	required_language = null
-
+/* We are commenting this out to avoid glitches with the skrell modpack
```

Comment:
Don't comment code out, just remove it. We can look back in the Git history and readd it.

Reply:
Done.